atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -58,10 +58,7 @@ public MaskedValue getDbPassword() {
 
     public abstract String getDriverClass();
 
-    @Value.Default
-    public String getTestQuery() {
-        return "";
-    }
+    public abstract Optional<String> getTestQuery();
```

Comment:
This will cause an API break 🔥 though an internal code search found that the only user of this is our internal implementation of the timelock service. I think we can accept this break, but @Sam-Kramer we'll likely need to switch that to deal correctly with this type change next week